Command/AbstractGenerateCommand.php

```diff
@@ -70,7 +103,12 @@ protected function checkOutputDirExists($outputDir)
     {
```

Comment:
[minor] should renamed to `createOutputDir` or smth like that, because is not a simple check anymore.